about.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Conference Page</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="responsive-style.css">
+  <script src="mobileMenu.js" defer></script>
+</head>
+<body>
+  <section id="home">
+    <nav>
+      <a href="index.html"><img src="img/logo.png" alt="" class="cclogo"></a>
+      <img src="img/mobile_menu.png" alt="" class="hamburger">
+      <ul class="link-items">
+        <li><a href="about.html">About</a> <img src="img/close_mobile_menu.png" alt=""></li>
+        <li><a href="index.html">Program</a></li>
+        <li><a href="index.html">Join</a></li>
+        <li><a href="index.html">Sponsor</a></li>
+        <li><a href="index.html">News</a></li>
+        <li><a href="index.html">CC Campaign</a></li>
+      </ul>
```

Comment:
- **[OPTIONAL]** When we are on the about page we need a link to the home page, please 🙏🏽 add it in the desktop and mobile version for a better user experience 👇🏽 
![image](https://user-images.githubusercontent.com/58642949/211154551-eaaab565-82d8-4126-ada1-00c3f7f220c0.png)